pages/mydoc/notes.md

```diff
@@ -1,13 +1,38 @@
 ---
 title: "Discussion Notes"
-keywords: icedb libicedb
+keywords: icedb libicedb discussion notes
 tags: [collaboration]
 sidebar: mydoc_sidebar
 permalink: notes.html
 summary: Short summaries of the web meeting among the development team
 ---
 
-## 17 April 2018
+Since the last IPWG/IWSSM (Bologna 2016) meeting a couple of people (Stefan, Jussi, Davide, Eugene, Ben) worked on an excel-sheet with an initial file structure and variable names (following as closely as possible the CF convention) for storing scattering datasets. The Excel-file can be downloaded [here]
```

Comment:
The Excel link needs to point somewhere. These sheets are already uploaded on the website, so perhaps https://rhoneyager.github.io/libicedb/structs_metadata.html?